ap.md

```diff
@@ -0,0 +1,256 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+<div class="large-12 columns">
+# Build Your Own Internet
+</div>
+<div class="large-7 small-12 columns">
+
+As explained in [wifi tutorial](/wifi.html), Tessel 2 can easily connect to wifi networks using the command line tool. If we're using a device connected to the same network as the Tessel, then we can connect to it through our device's web browser. Like most network-connected devices, communication over the web depends on a shared router, or access point, to direct requests and responses between every device on the network or through the Internet. This is fine and dandy if we want to use Tessel in a place with an available network, but what happens when that's not possible? 
+
+</div>
+<div class="large-5 small-12 columns right">
+![router](./images/router.png)
+</div>
+<div class="large-7 small-12 columns">
+Instead of requiring a shared network through a standalone router, we can use the command line tool for configuring Tessel to emit a custom Wifi network and use it as an access point. Then, using Node, we can also start a web server for communicating with any connected device. Unfortunately, we won't be able to connect to the Internet but there is still a lot we can do without that. By the end of this article, we'll be able to control the Tessel LEDs through a web app served by the Tessel.
```

Comment:
Absolutely brilliant. I really like where you broke it down and how you explained the typical wireless setup. :+1: I'll fix up the introduction with this.